README.md

```diff
@@ -10,13 +10,27 @@ Other platforms:
 
 ## Prerequisites
 - The latest stable version of [Xcode](https://apps.apple.com/us/app/xcode/id497799835)
-- The latest stable version of [CocoaPods](https://cocoapods.org/)
+- The latest stable version of [CocoaPods](https://cocoapods.org/) (not needed when using Swift Package Manager)
 - Sign up to [Whereby Embedded](https://whereby.com/information/embedded/) account
 - [Create a room](https://docs.whereby.com/creating-and-deleting-rooms) in your Whereby Embedded account
 
 Whereby SDK supports iOS 14.0 as minimum deployment target.
 
 ## Installation
+
+### Using Swift Package Manager
+
+1. Open your project in Xcode and select File > Swift Packages > Add Package Dependency.
```

Comment:
Minor detail: in my case, it shows as `Add Packages` instead of `Swift Packages`. I'm using XCode 13.4.1

Reply:
Fixed. I just copied it from Apple's doc initially so they have it outdated 🤷‍♂️